### ISSUE TITLE
AWS: can not sign up

### ISSUE THREAD

**Issue by natoucs** [HPCE/hpce-2017-cw4#28]
I signed up to AWS with my imperial email via this link on monday at 10am: https://www.awseducate.com/registration
I then received a link to confirm my email and did so.
Then I received this email:

`Hi Nathan,
Thank you for applying for AWS Educate. We have received your application, and it is currently under review. You will receive an email once the review is complete.
If you have any questions, please click here to contact AWS Educate support.
Thank You! 
The AWS Educate Team`

I still have not heard back from them to start setting up my account.
When I try to log in, it says my email is not registered yet (which is normal).

Everybody I spoke to told me their sign up was instantaneous.

Any solutions to bypass this situation ?

**Comment by JB515** [comment]
I had this but got an email saying I'd been approved the following day. Can't find any way to actually start an instance direct from awseducate without giving card details to aws, but that's a separate issue...

**Comment by natoucs** [comment]
Received acceptation email this morning, took 2 days, be careful if you start the CW late!

**Comment by natoucs** [comment]
Actually I just got access to aws educate, not to the aws.amazon where it seems my account does not exist. So still an issue at the moment.
